slybot/slybot/plugins/scrapely_annotations/annotations.py

```diff
@@ -83,15 +86,20 @@ def setup_bot(self, settings, spec, items, extractors):
 
         # generate ibl extractor for links pages
         _links_pages = [dict_to_page(t, 'annotated_body')
-                        for t in spec['templates']
-                        if t.get('page_type') == 'links']
+                        for t in templates if t.get('page_type') == 'links']
         _links_item_descriptor = create_slybot_item_descriptor({'fields': {}})
         self._links_ibl_extractor = InstanceBasedLearningExtractor(
             [(t, _links_item_descriptor) for t in _links_pages]) \
             if _links_pages else None
 
         self.build_url_filter(spec)
 
+    def _get_annotated_template(self, template):
+        if template.get('version', '0.12.0') >= '0.13.0':
+            if 'annotated_body' not in template:
```

Comment:
It should build the sample even if annotated body is present as `annotated_body` is not updated in portia 2.0